e2e/fixtures/auth.ts

```
@@ -0,0 +1,87 @@
+    const user = await createTestUser(email);
+    
+    // Generate token and perform auto-login
+    const token = await generateAutoLoginToken(user.email);
+    
+    // Visit auto-login URL
+    await page.goto(`/api/auth/auto-login?token=${token}`);
+    
+    // Wait for redirect to home page
+    await page.waitForURL('/');
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)